scripts/zones/Jugner_Forest/mobs/Meteormauler_Zhagtegg.lua

```diff
@@ -2,13 +2,57 @@
 -- Area: Jugner Forest
 --   NM: Meteormauler Zhagtegg
 -----------------------------------
+mixins = {require("scripts/mixins/job_special")}
+local ID = require("scripts/zones/Jugner_Forest/IDs")
+-----------------------------------
 local entity = {}
 
+-- TODO: Implement better pathing systems for guards to follow master
+
+entity.onMobSpawn = function(mob)
+    -- Takes half damage from all attacks
+    mob:addMod(xi.mod.UDMGPHYS,-5000)
```

Comment:
some things:

If cutting all damage types, there is a single mod that can be used for all of these. It doesn't have to be the uncapped version of each separately since mobs do not cap below 50% DT. BG specifies it as 50% magic damage, while clopedia doesn't.

Reply:
Got that adjusted to just one mod